FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -66,12 +73,26 @@ @implementation FPRAppActivityTracker
 + (void)load {
   // This is an approximation of the app start time.
   appStartTime = [NSDate date];
+
+  if (![[[NSBundle mainBundle] objectForInfoDictionaryKey:@"fireperf_disable_dd"] boolValue]) {
```

Comment:
Are we just checking for an entry in the PLIST or are we checking for the value of the key in the plist?

Reply:
Current implementation checks for value. The value must be a `boolean` `1`. Checking for just entry is an alternative, what would be best for this case?

Reply:
Since we are checking for a bool value, I would say lets check for the booValue to return to "Yes".